internal/pkg/client/gitlab/client.go

```diff
@@ -70,5 +138,14 @@ func (client *Client) Parse(rawURL string) *netutil.URL {
 		Port:     results["port"],
 		Path:     results["path"],
 		GitRef:   results["gitRef"],
+		Query:    results["query"],
 	}
 }
+
+func validateHTTPStatusCode(statusCode int) error {
+	if statusCode < http.StatusOK || http.StatusMultipleChoices <= statusCode {
```

Comment:
nit: slightly more readable to me (subjective). Did the linter suggest this?

```suggestion
	if statusCode < http.StatusOK || statusCode >= http.StatusMultipleChoices {
```

Reply:
All set. The linter had no suggestions on comparison order.